client/webserver/site/src/js/markets.ts

```diff
@@ -241,7 +241,13 @@ export default class MarketsPage extends BasePage {
 
     // Prepare templates for the buy and sell tables and the user's order table.
     Doc.cleanTemplates(page.rowTemplate, page.liveTemplate, page.durBttnTemplate, page.booleanOptTmpl, page.rangeOptTmpl, page.orderOptTmpl)
-    OrderUtil.setOptionTemplates(page)
+
+    const setOrderOptionTemplates = () => {
+      OrderUtil.setRangeOptionTempalate(page.rangeOptTmpl)
+      OrderUtil.setBooleanOptionTemplate(page.booleanOptTmpl)
+      OrderUtil.setOrderOptionTempalate(page.orderOptTmpl)
+    }
+    setOrderOptionTemplates()
```

Comment:
Don't need to separate these if you're not changing them (which you shouldn't be). Just pull out the new template in `setOptionTemplates`.